### PR TITLE
feat(mojaloop/#3436): nodejs upgrade

### PR DESCRIPTION
**Pull request by aaronreynoza**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -5,6 +5,15 @@ const HapiOpenAPI = require('hapi-openapi')
 const Path = require('path')
 const Logger = require('@mojaloop/central-services-logger')
 
+module.exports.stopServer = async function (server) {
+  try {
+    await server.stop()
+    console.log('Server stopped')
```
Comment (geka-evk) [inline src/server.js:11]: Why do we use  _console.log_ here, and not Logger?

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,29 @@
-FROM node:16.15.0-alpine as builder
+# Arguments
+ARG NODE_VERSION=lts-alpine
+
+# NOTE: Ensure you set NODE_VERSION Build Argument as follows...
+#
+#  export NODE_VERSION="$(cat .nvmrc)-alpine" \
+#  docker build \
+#    --build-arg NODE_VERSION=$NODE_VERSION \
+#    -t mojaloop/sdk-scheme-adapter:local \
```
Comment (geka-evk) [inline Dockerfile:9]: Maybe, it's better to update image name according repository name?

**Review by geka-evk — APPROVED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -596,6 +699,28 @@ workflows:
...
+      #     context: org-global
+      #     requires:
+      #       - setup
+      #     filters:
+      #       tags:
+      #         only: /.*/
+      #       branches:
+      #         ignore:
+      #           - /feature*/
+      #           - /bugfix*/
```
Comment (oderayi) [inline .circleci/config.yml:723]: This can be removed until we have integration and functional tests.

File: .circleci/config.yml [hunk 4]
```diff
@@ -622,24 +747,13 @@ workflows:
...
-      - build:
-          context: org-global
-          requires:
-            - setup
-            - test-dependencies
             - test-lint
             - test-unit
             - test-coverage
+            # - test-integration
+            # - test-functional
```
Comment (oderayi) [inline .circleci/config.yml:754]: Can be removed.

File: src/server.js [hunk 5]
```diff
@@ -5,6 +5,15 @@ const HapiOpenAPI = require('hapi-openapi')
 const Path = require('path')
 const Logger = require('@mojaloop/central-services-logger')
 
+module.exports.stopServer = async function (server) {
```
Comment (oderayi) [inline src/server.js:8]: Thanks for adding this and the test.

**Review by oderayi — CHANGES_REQUESTED** [review]
Looking good.

- Please do snapshot release to validate the release pipeline.

**Review by oderayi — APPROVED** [review]

**Pull request merged**